agent/cleanup.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -x
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+LOGDIR=${SCRIPTDIR}/logs
+source $SCRIPTDIR/logging.sh
+source $SCRIPTDIR/common.sh
+source $SCRIPTDIR/utils.sh
+source $SCRIPTDIR/validation.sh
+source $SCRIPTDIR/agent/common.sh
+
+early_cleanup_validation
```

Comment:
Is this destroying and undefining the libvirt domains?

Reply:
No, that's in the `host_cleanup` target